docs/logs/getting-started-aspnetcore/README.md

```diff
@@ -35,7 +35,7 @@ LogRecord.SeverityText:            Information
 LogRecord.Body:                    Starting the app...
 LogRecord.Attributes (Key:Value):
     OriginalFormat (a.k.a Body): Starting the app...
-LogRecord.EventId:                 1
+LogRecord.EventId:                 225744744
```

Comment:
ohh! "magic" commit was this, I suppose!

![image](https://github.com/open-telemetry/opentelemetry-dotnet/assets/5232798/9a03e731-0fca-471c-8e90-f85036161f96)

Reply:
Yup. https://github.com/open-telemetry/opentelemetry-dotnet/pull/4821#discussion_r1316757504

Reply:
That is good to know! 
I think we need to mention that here, and call out that it is only if using v9 of m.e.logging.